runtime/sqldelight-runtime/build.gradle

```diff
@@ -45,8 +45,12 @@ kotlin {
   targets {
     fromPreset(presets.jvm, 'jvm')
     fromPreset(presets.js, 'js')
-    fromPreset(presets.iosX64, 'iosX64')
-    fromPreset(presets.iosArm64, 'iosArm64')
+    fromPreset(presets.iosX64, 'iosX64') {
+      compilations.main.outputKinds('KLIBRARY')
```

Comment:
I think you just want `LIBRARY`: https://github.com/JetBrains/kotlin/blob/2251440f04e5bdb4bdfef2dc47c30356b7f39411/konan/utils/src/org/jetbrains/kotlin/konan/target/KonanTarget.kt#L70